gitlab2sentry/utils/gitlab_provider.py

```diff
@@ -221,7 +221,10 @@ def _get_default_mentions(self, project: Project) -> str:
             [
                 f"@{member.username}"
                 for member in project.members.all()
-                if member.access_level >= GITLAB_MENTIONS_ACCESS_LEVEL
+                if (
+                    member.access_level >= GITLAB_MENTIONS_ACCESS_LEVEL
+                    and member.state != "blocked"
```

Comment:
Maybe we could have an Enum instead of a hard coded string here (and also in the tests) ?

Reply:
@rcollas yes, that should be proposed as a separate PR please?

Reply:
Hey @ultrabug & @rcollas thanks for the review <3

Good catch for the hardcoded string! I don't have merge perms so feel free to merge!